### PR TITLE
✨Source Pivotal-Tracker: Migrate Python CDK to Low-code CDK

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
*Migrating Source Pivotal Tracker to Low-Code CDK*
Closes airbytehq/airbyte-internal-issues#7010

## How
*Developed using (Configuration Based Source) low-code CDK*

## Recommended reading order
1. `spec.yaml`
2. `manifest.yaml`
3. `schemas/*`

## Tests

<details open><summary><strong>Integration & Acceptance</strong></summary>

 - Full passed state:
 
image

</details>

## 🚨 User Impact 🚨
- Migration to low-code

### CODE REVIEW

**Comment by btkcodedev** [comment]
Inline Schema ✅
Poetry lock file ✅
No Breaking change ✅
Docs updated ✅

`airbyte-ci test` All pass state:
Results:
image

**Comment by btkcodedev** [comment]
Marking PR as ready to merge

File: airbyte-integrations/connectors/source-pivotal-tracker/metadata.yaml [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
-    oss:
-      enabled: true
+  releaseDate: 2022-04-04
   releaseStage: alpha
+  supportLevel: community
   documentationUrl: https://docs.airbyte.com/integrations/sources/pivotal-tracker
   tags:
     - language:python
-    - cdk:python
-  ab_internal:
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/metadata.yaml:25]: Why did you delete this section?

File: airbyte-integrations/connectors/source-pivotal-tracker/pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+python = "^3.9,<3.12"
+airbyte-cdk = "^0"
+
+[tool.poetry.scripts]
+source-pivotal-tracker = "source_pivotal_tracker.run:run"
+
+[tool.poetry.group.dev.dependencies]
+requests-mock = "*"
+pytest-mock = "*"
+pytest = "*"
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/pyproject.toml:27]: Please add an empty line at the bottom of the file

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_source.py [hunk 3]
```diff
@@ -1,25 +0,0 @@
...
-#
-
-from unittest.mock import MagicMock
-
-import responses
-from source_pivotal_tracker.source import SourcePivotalTracker
-
-
-@responses.activate
-def test_check_connection(config_pass, projects_response):
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_source.py:12]: These tests should not be deleted. Instead, they should be aligned with the new approach

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py [hunk 4]
```diff
@@ -1,102 +0,0 @@
...
-from airbyte_cdk.models import SyncMode
-from source_pivotal_tracker.source import Activity, Epics, Labels, PivotalAuthenticator, ProjectMemberships, Projects, Releases, Stories
-
-auth = PivotalAuthenticator("goodtoken")
-project_args = {"project_ids": [98, 99], "authenticator": auth}
-stream_slice = {"project_id": 99}
-
-
-@responses.activate
-def test_projects_stream(projects_response):
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py:15]: These tests should not be deleted. Instead, they should be aligned with the new approach

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml [hunk 5]
```diff
@@ -0,0 +1,1609 @@
...
+                      type: CursorPagination
+                      cursor_value: >-
+                        {{ response['headers',
+                        'X-Tracker-Pagination-Offset']+response['headers',
+                        'X-Tracker-Pagination-Limit' ]}}
+                      stop_condition: >-
+                        {{ 'X-Tracker-Pagination-Total' not in
+                        response['headers'] }}
+  - type: DeclarativeStream
+    name: activity
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:1121]: Missing transformation from the Python implementation - please add as described [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/source.py#L103)

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml [hunk 6]
```diff
@@ -0,0 +1,1609 @@
...
+            type: number
+          secondary_resources:
+            type: array
+        type: object
+    retriever:
+      type: SimpleRetriever
+      requester:
+        type: HttpRequester
+        url_base: https://www.pivotaltracker.com/services/v5/
+        path: projects/{{ stream_slice['project_id'] }}/activity?limit=10
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:1433]: The `path` parameter should not contain query params. Please handle this via pagination. In addition, this stream has less records than the original one.

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml [hunk 7]
```diff
@@ -0,0 +1,1609 @@
...
+          updated_at:
+            type: string
+          velocity_averaged_over:
+            type: number
+          version:
+            type: number
+          week_start_day:
+            type: string
+        type: object
+    retriever:
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:85]: The `retriever` parameter seems to be the same for all streams - could you please declare it separately and re-use?

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/components.py [hunk 8]
```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from typing import Any, List, Mapping, Optional, Tuple
+
+from airbyte_cdk.sources.declarative.requesters.paginators.strategies.page_increment import PageIncrement
+
+
+class ItemCursorPaginationStrategy(PageIncrement):
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/components.py:10]: What is the purpose of this class?

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml [hunk 9]
```diff
@@ -0,0 +1,1609 @@
...
+          type: RequestOption
+          inject_into: request_parameter
+          field_name: offset
+        pagination_strategy:
+          type: CursorPagination
+          cursor_value: >-
+            {{ response['headers',
+            'X-Tracker-Pagination-Offset']+response['headers',
+            'X-Tracker-Pagination-Limit' ]}}
+          stop_condition: "{{ 'X-Tracker-Pagination-Total' not in response['headers'] }}"
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:119]: Should [this](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/source.py#L32) condition also be included here?

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml [hunk 10]
```diff
@@ -0,0 +1,1609 @@
...
+          type: DpathExtractor
+          field_path: []
+      paginator:
+        type: DefaultPaginator
+        page_token_option:
+          type: RequestOption
+          inject_into: request_parameter
+          field_name: offset
+        pagination_strategy:
+          type: CursorPagination
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:114]: Should the `OffsetIncrement` strategy be used here?

**Review by askarpets — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_source.py (hunk 3 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_source.py:12]: Could you please eloborate this process?
Low code cdk usually perform tests within standard acceptance-test yaml file isnt it?

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py (hunk 4 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py:15]: @askarpets Could you please eloborate this process?
Low code cdk usually perform tests within standard acceptance-test yaml file isnt it?

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml (hunk 6 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:1433]: It is a little workaround, as cursor Pagination field from response gives full next page path, it is directly injected to path
The newly injected path should not contain any limit parameter.

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml (hunk 10 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:114]: It is Cursor Pagination as described in docs and previous python version, the response gives next link and it is injected to path

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml (hunk 6 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:1433]: The Pagination is properly handled via Cursor Pagination block

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py (hunk 4 above)
Comment (askarpets) [reply airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py:15]: You can retrieve a stream needed for a test from the source instead of importing its class. Please see Gitlab [implementation](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-gitlab/unit_tests/test_streams.py)

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_source.py (hunk 3 above)
Comment (askarpets) [reply airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_source.py:12]: We should keep as many unit tests as possible after migration. You should use migrated source in these tests

Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_source.py:12]: For this specific scenario of pivotal-tracker, the `connection check` and `streams check` are already done via standard `acceptance-test-config.yaml` file itself, these world be duplicate checks for this connector
As the gitlab connector has stream slices and oauth config for additional checks, it might require the unit tests but it would be not required here, would be my suggestion

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py (hunk 4 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py:15]: For this specific scenario of pivotal-tracker, the connection check and streams check are already done via standard acceptance-test-config.yaml file itself, these would be duplicate checks for this connector.
As the gitlab connector has stream slices and oauth config for additional checks, it might require the unit tests but it would be not required here, would be my suggestion

**Comment by btkcodedev** [comment]
All comments resolved

File: airbyte-integrations/connectors/source-pivotal-tracker/integration_tests/sample_state.json [hunk 11]
```diff
@@ -0,0 +1,5 @@
+{
+  "todo-stream-name": {
```
[the remark this answers is not in the document]
Comment (askarpets) [reply airbyte-integrations/connectors/source-pivotal-tracker/integration_tests/sample_state.json:2]: @btkcodedev could you please delete this file as well?

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml (hunk 5 above)
Comment (askarpets) [reply airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml]: @btkcodedev field `project_id` is not present in records

**Comment by askarpets** [comment]
@btkcodedev can you please fix the following schemas related issues?

`projects` stream:

- remove field `required`
- field `id` should have `type` of `["null", "integer"]`
- field `created_at` should have `format` of `"date-time"`
- field `updated_at` should have `format` of `"date-time"`
- field `project_id` should be added to the schema to avoid breaking changes during migration
- field `account_id` should have `type` of `["null", "integer"]`
- field `version` should have `type` of `["null", "integer"]`
- field `iteration_length` should have `type` of `["null", "integer"]`
- field `current_iteration_number` should have `type` of `["null", "integer"]`
- field `velocity_averaged_over` should have `type` of `["null", "integer"]`
- field `number_of_done_iterations_to_show` should have `type` of `["null", "integer"]`
- field `initial_velocity` should have `type` of `["null", "integer"]`
- field `start_time` should have `format` of `"date-time"`

`activity` stream:

- remove field `required`
- field `secondary_resources` should have `items` with `type` of `["null", "string"]`
- field `project_id` should be added as well as logic to add this field in the stream
- field `occurred_at` should have `format` of `"date-time"`
- field `performed_by.id` should have `type` of `["null", "integer"]`
- fields `performed_by.email` and `performed_by.username` should be added
- field `project_version` should have `type` of `["null", "integer"]`
- field `changes.items.id` should have `type` of `["null", "integer"]`
- field `primary_resources.items.id` should have `type` of `["null", "integer"]`

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_source.py (hunk 3 above)
Comment (askarpets) [reply airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_source.py:12]: @lazebnyi could you please take a look at the comment above and let us know if these tests should be kept?

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py (hunk 4 above)
Comment (askarpets) [reply airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py:15]: @lazebnyi could you please take a look at the comment above and let us know if these tests should be kept?

**Review by lazebnyi — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml (hunk 9 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml]: Paginated with Standard Offset pagination

**Review by askarpets — CHANGES_REQUESTED** [review]
@btkcodedev, streams' schemas look good now, thanks for the update!

Could you please also make the following changes?

- `integration_tests/sample_state.json` file should be deleted as this connector doesn't have streams with incremental sync
- Unit tests should be added back with corresponding alignments
- `project_id` field should be added to the `activity` stream as described [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/source.py#L103). Please use the `transformations` section at the stream level to have this implemented
- Please align the spec to fully match what was in the original version

Thank you!

**Comment by btkcodedev** [comment]
Changes done :white_check_mark: 
CI - Passing

**Comment by btkcodedev** [comment]
@askarpets All comments resolved and CI is passing, please continue the merge process :bow:

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py [hunk 12]
```diff
@@ -2,101 +2,133 @@
...
 
-import responses
+import json
+import os
+from typing import Any, Mapping
+
 from airbyte_cdk.models import SyncMode
-from source_pivotal_tracker.source import Activity, Epics, Labels, PivotalAuthenticator, ProjectMemberships, Projects, Releases, Stories
+from airbyte_cdk.sources.streams import Stream
+from pytest import fixture
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py:11]: Please remove this import as it's not used here

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py [hunk 13]
```diff
@@ -2,101 +2,133 @@
...
-    responses.add(
-        responses.GET,
-        "https://www.pivotaltracker.com/services/v5/projects/99/stories",
-        json=stories_response,
-    )
-    stream = Stories(**project_args)
-    records = [r for r in stream.read_records(SyncMode.full_refresh, None, stream_slice, None)]
-    assert len(records) == 4
-    assert len(responses.calls) == 1
+def load_file(fn):
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py:19]: Please delete this function and use the one from `conftest.py` to avoid code duplicates

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py [hunk 14]
```diff
@@ -2,101 +2,133 @@
...
+
+def get_stream_by_name(stream_name: str, config: Mapping[str, Any]) -> Stream:
+    source = SourcePivotalTracker()
+    matches_by_name = [stream_config for stream_config in source.streams(config) if stream_config.name == stream_name]
+    if not matches_by_name:
+        raise ValueError("Please provide a valid stream name.")
+    return matches_by_name[0]
+
+
+projects_mock = json.loads(load_file("projects.json"))
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py:31]: Please use existing fixtures from the `conftest.py` for all tests in this module and delete these mocks

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py [hunk 15]
```diff
@@ -2,101 +2,133 @@
...
+def test_projects(requests_mock):
+    requests_mock.get(url="https://www.pivotaltracker.com/services/v5/projects", status_code=200)
+    stream = get_stream_by_name("projects", CONFIG)
+    expected_record = dict(projects_mock[0])
+    for json in projects_mock:
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py:36]: Please mock the data in one place instead of having this loop (applies to all tests here)

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py [hunk 16]
```diff
@@ -2,101 +2,133 @@
...
+    requests_mock.get(url="https://www.pivotaltracker.com/services/v5/projects/99/labels", status_code=200)
+    stream = get_stream_by_name("labels", CONFIG)
+    expected_record = dict(labels_mock[0])
+    for json in projects_mock:
+        requests_mock.get("project", json=json)
+    for json in labels_mock:
+        requests_mock.get("labels", json=json)
+    for stream_slice in stream.stream_slices(sync_mode=SyncMode.full_refresh):
+        for record in stream.read_records(sync_mode=SyncMode.full_refresh, stream_slice=stream_slice):
+            count += 1
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py:88]: What is the purpose of this variable?

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml [hunk 17]
```diff
@@ -0,0 +1,876 @@
...
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: projects/{{ stream_slice['project_id'] }}/activity
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+        paginator:
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:188]: Please check how the pagination was implemented [originally](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/source.py#L22) and implement it in the same way for all streams here

**Review by askarpets — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml (hunk 17 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:188]: It is already implemented as standard pagination and verified its working

Comment (askarpets) [reply airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:188]: Shouldn't these info be taken from headers?

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py (hunk 16 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py]: Removed

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py (hunk 12 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py]: Done

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py (hunk 15 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py]: Done

File: airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py (hunk 13 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py]: Done

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml (hunk 17 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:188]: It finally comes down to standard offset pagination, I've verified it paginating in the builder itself and via test
image

**Comment by btkcodedev** [comment]
Tests fixed :white_check_mark:

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml [hunk 18]
```diff
@@ -0,0 +1,876 @@
...
+            type: RequestOption
+            inject_into: request_parameter
+            field_name: offset
+          page_size_option:
+            type: RequestOption
+            field_name: limit
+            inject_into: request_parameter
+          pagination_strategy:
+            type: OffsetIncrement
+            page_size: 10
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:200]: I think this parameter should be taken from the response - please see [doc](https://www.pivotaltracker.com/help/api#Paginating_List_Responses). Also, this param should not be included on the first call

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml [hunk 19]
```diff
@@ -0,0 +1,876 @@
...
+              - type: ParentStreamConfig
+                parent_key: id
+                partition_field: project_id
+                stream:
+                  $ref: "#/definitions/streams/projects"
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/labels"
+    releases:
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:116]: Pagination should be added for this stream according to the [doc](https://www.pivotaltracker.com/help/api/rest/v5#Releases)

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml [hunk 20]
```diff
@@ -0,0 +1,876 @@
...
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/projects"
+    stories:
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:32]: Pagination should be added for this stream according to the [doc](https://www.pivotaltracker.com/help/api/rest/v5#Releases)

**Review by askarpets — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml (hunk 18 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:200]: the page_size is integer type variable and cannot be changed with jinja macros in Yaml
Even if we try cursor pagination, for getting the value, the page_size is strictly typed
image

Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:200]: Is there any neccesity for changing this? Current pagination strategy works perfectly
Ref: Builder - Partition 2
image

Builder - Partition 1
image

Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:200]: Docs says ``
image
`If no limit parameter is supplied for a paginated request, a default limit value will be used`

File: airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml (hunk 19 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:116]: Labels doesn't support pagination,

Comment (btkcodedev) [reply airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml:129]: Added for stories

**Comment by btkcodedev** [comment]
Comments resolved, Please continue merging process

**Review by askarpets — APPROVED** [review]

**Review by lazebnyi — APPROVED** [review]
